### PR TITLE
Add back pointerlock{error|change} events

### PR DESCRIPTION
**Pull request by Elchi3**
#### Summary

https://github.com/mdn/browser-compat-data/pull/16674 removed the `pointerlockchange` and `pointerlockerror` events from `api/Document.json`. 
I think the idea was to move them to `api/Element.json`, so this PR adds them back to `Element`.

#### Test results and supporting details

https://developer.mozilla.org/en-US/docs/Web/API/Document/pointerlockerror_event and
https://developer.mozilla.org/en-US/docs/Web/API/Document/pointerlockchange_event have broken compat tables (and the pages need to move to Element).

---
MDN content PR is https://github.com/mdn/content/pull/20806

### CODE REVIEW

**Comment by Elchi3** [comment]
@foolip given https://github.com/mdn/browser-compat-data/pull/16674#pullrequestreview-1028330809 and https://github.com/mdn/browser-compat-data/issues/17821 can you say if this should live under `Element` or under `Document`?

**Comment by foolip** [comment]
cc @queengooborg

This was a mishap in https://github.com/mdn/browser-compat-data/pull/16674. `npm run diff` shows these changes:

```
api.Document:
 → api.Document.pointercancel_event is deleted
 → api.Document.pointerdown_event is deleted
 → api.Document.pointerenter_event is deleted
 → api.Document.pointerleave_event is deleted
 → api.Document.pointerlockchange_event is deleted
 → api.Document.pointerlockerror_event is deleted
 → api.Document.pointermove_event is deleted
 → api.Document.pointerout_event is deleted
 → api.Document.pointerover_event is deleted
 → api.Document.pointerrawupdate_event is deleted
 → api.Document.pointerup_event is deleted
api.Element:
 → api.Element.pointercancel_event is added
 → api.Element.pointerdown_event is added
 → api.Element.pointerenter_event is added
 → api.Element.pointerleave_event is added
 → api.Element.pointermove_event is added
 → api.Element.pointerout_event is added
 → api.Element.pointerover_event is added
 → api.Element.pointerrawupdate_event is added
 → api.Element.pointerup_event is added
api.GlobalEventHandlers:
 → api.GlobalEventHandlers.onpointercancel is deleted
 → api.GlobalEventHandlers.onpointerdown is deleted
 → api.GlobalEventHandlers.onpointerenter is deleted
 → api.GlobalEventHandlers.onpointerleave is deleted
 → api.GlobalEventHandlers.onpointermove is deleted
 → api.GlobalEventHandlers.onpointerout is deleted
 → api.GlobalEventHandlers.onpointerover is deleted
 → api.GlobalEventHandlers.onpointerrawupdate is deleted
 → api.GlobalEventHandlers.onpointerup is deleted
api.HTMLElement:
 → api.HTMLElement.pointercancel_event is deleted
 → api.HTMLElement.pointerdown_event is deleted
 → api.HTMLElement.pointerenter_event is deleted
 → api.HTMLElement.pointerleave_event is deleted
 → api.HTMLElement.pointermove_event is deleted
 → api.HTMLElement.pointerout_event is deleted
 → api.HTMLElement.pointerover_event is deleted
 → api.HTMLElement.pointerrawupdate_event is deleted
 → api.HTMLElement.pointerup_event is deleted
```

These two events were deleted, but not added anywhere else. Everything else lines up, so this PR should sort it out.

**Comment by foolip** [comment]
`Document` is the right event target here per spec:
https://w3c.github.io/pointerlock/#pointerlockchange-and-pointerlockerror-events

I've also checked Chromium source to confirm the event ends up fired on `Document`.

File: api/Element.json [hunk 1]
```diff
@@ -6183,6 +6183,136 @@
           }
         }
       },
+      "pointerlockchange_event": {
```
Comment (foolip) [inline api/Element.json:6186]: Can you move to `Document`?

File: api/Element.json [hunk 2]
```diff
@@ -6183,6 +6183,136 @@
...
+            "webview_android": [
+              {
+                "version_added": "37"
+              },
+              {
+                "version_added": "≤37",
+                "version_removed": "38",
+                "prefix": "webkit"
+              }
+            ]
```
Comment (foolip) [inline api/Element.json:6242]:
```suggestion
            "webview_android": "mirror"
```

File: api/Element.json [hunk 3]
```diff
@@ -6183,6 +6183,136 @@
...
```
Comment (foolip) [inline api/Element.json:6307]:
```suggestion
            "webview_android": "mirror"
```

**Review by foolip — CHANGES_REQUESTED** [review]

**Comment by foolip** [comment]
As an aside, https://github.com/w3c/pointerlock/issues/21 is a suggestion in the spec to fire the event on `Element` instead, but that was never changed.

**Comment by Elchi3** [comment]
Updated. Fixing the content page, too.

**Review by foolip — APPROVED** [review]

**Comment by Elchi3** [comment]
Content PR was merged. Merging this, too.

**Pull request merged**